test/parallel/test-event-emitter-max-listeners.js

```diff
@@ -11,14 +11,14 @@ e.setMaxListeners(42);
 
 assert.throws(function() {
   e.setMaxListeners(NaN);
-});
+}, Error);
```

Comment:
These checks would be more useful with a regexp as the second argument rather than simply `Error`:

```js
assert.throws(function() {
  e.setMaxListeners(NaN);
}, /^TypeError: "n" argument must be a positive number$/);
```